### PR TITLE
[10.2] Remove references to the Gallery app (#1607)

### PR DESCRIPTION
**Pull request by settermjd**
This relates to #1606 and #1515.

### CODE REVIEW

File: modules/admin_manual/pages/configuration/files/encryption/encryption_configuration.adoc [hunk 1]
```diff
@@ -416,7 +416,6 @@ the filenames or folder structures. These files are never encrypted:
 * Existing files in the trash bin & Versions. Only new and changed files
 after encryption is enabled are encrypted.
 * Existing files in Versions
-* Image thumbnails from the Gallery app
```
Comment (pmaier1) [inline modules/admin_manual/pages/configuration/files/encryption/encryption_configuration.adoc]: Image thumbnails are independent of the Gallery app. I think I wrote this in another PR already: Please just remove "from the Gallery app".

Question: Why is this same text written in three different files?

File: modules/admin_manual/pages/configuration/files/encryption/index.adoc [hunk 2]
```diff
@@ -114,7 +114,6 @@ The following files are *never* encrypted:
 +
 NOTE: You can post encrypt existing files via an xref:configuration/server/occ_command.adoc#encryption[occ encryption command].
 * Existing files in Version
-* Image thumbnails from the Gallery app
```
Comment (pmaier1) [inline modules/admin_manual/pages/configuration/files/encryption/index.adoc]: Image thumbnails are independent of the Gallery app. I think I wrote this in another PR already: Please just remove "from the Gallery app".

Question: Why is this same text written in three different files?

File: modules/admin_manual/pages/configuration/files/previews_configuration.adoc [hunk 3]
```diff
@@ -4,7 +4,7 @@
 == Introduction
 
 The ownCloud thumbnail system generates previews of files for all
-ownCloud apps that display files, such as Files and Gallery.
+ownCloud apps that display files, such as Files.
```
Comment (pmaier1) [inline modules/admin_manual/pages/configuration/files/previews_configuration.adoc]: ... and Media Viewer ?

File: modules/user_manual/pages/files/encrypting_files.adoc [hunk 4]
```diff
@@ -87,7 +87,6 @@ Only the data in your files is encrypted, and not the filenames or
 folder structures. These files are never encrypted:
 
 * Old files in the trash bin.
-* Image thumbnails from the Gallery app.
```
Comment (pmaier1) [inline modules/user_manual/pages/files/encrypting_files.adoc]: Image thumbnails are independent of the Gallery app. I think I wrote this in another PR already: Please just remove "from the Gallery app".

Question: Why is this same text written in three different files?

File: modules/admin_manual/pages/configuration/files/encryption/encryption_configuration.adoc (hunk 1 above)
Comment (settermjd) [reply modules/admin_manual/pages/configuration/files/encryption/encryption_configuration.adoc]: If I understand your intent correctly, I'll move the text to an includeable file.

File: modules/user_manual/pages/files/encrypting_files.adoc (hunk 4 above)
Comment (settermjd) [reply modules/user_manual/pages/files/encrypting_files.adoc]: I'll move the text to a separate, includable, partial.

File: modules/admin_manual/pages/configuration/files/encryption/index.adoc (hunk 2 above)
Comment (settermjd) [reply modules/admin_manual/pages/configuration/files/encryption/index.adoc]: I'll move the repeated text to a separate, includable, partial.

**Comment by settermjd** [comment]
#1839 needs to be merged to the 10.2 branch as part of this PR.

**Pull request merged**